src/components/DetailViewHeader.jsx

```diff
@@ -110,20 +141,57 @@ export class DetailViewHeader extends React.PureComponent {
                 canManageUsers={true}
               />
             )}
-            <Button className={classes.exportButton} variant="contained" color="primary">
-              <a
-                target="_blank"
-                rel="noopener noreferrer"
-                href={`${terraUrl}/#import-data?url=${window.location.origin}&snapshotId=${of.id}&snapshotName=${of.name}&format=snapshot`}
+            <hr className={classes.separator} />
+            <Typography variant="h6" className={classes.section}>
+              Export a copy of the snapshot metadata to an exisiting or new Terra workspace
+            </Typography>
+            {!isProcessing && !isDone && (
+              <TerraTooltip title="Exporting a snapshot to a workspace means that all members of your workspace will be able to have read only access to the tables and files in the snapshot">
+                <Button
```

Comment:
I noticed that the button takes the whole width.  Is that ok?  Not a huge deal

Reply:
I'm checking w/ Lou. I needed to make the button a bit bigger anyway to accommodate the spin/preparing snapshot text, so I thought I would just match the other buttons in the panel.